app/main.py

```diff
@@ -1,3 +1,56 @@
+from __future__ import annotations
+
+
 class Distance:
-    # Write your code here
-    pass
+
+    def __init__(self, km: int | float) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: Distance | int | float) -> Distance:
+        return Distance(self.km + other.km
```

Comment:
It is better to write it in like in checklist Imho 
![image](https://github.com/mate-academy/py-distance-class/assets/102595649/3d90af42-e3ed-47e6-9d34-ee6326a6667a)